tests/helper/helper_interactive.go

```diff
@@ -0,0 +1,66 @@
+// +build linux darwin dragonfly solaris openbsd netbsd freebsd
+
+package helper
+
+import (
+	"bytes"
+	"log"
+	"os/exec"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/kr/pty"
+	. "github.com/onsi/gomega"
+)
+
+func RunInteractive(commonVar CommonVar, command []string, test func(*expect.Console, *bytes.Buffer)) (string, error) {
+
+	ptm, pts, err := pty.Open()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	term := vt10x.New(vt10x.WithWriter(pts))
+
+	c, err := expect.NewConsole(expect.WithStdin(ptm), expect.WithStdout(term), expect.WithCloser(pts, ptm))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer c.Close()
+
+	// execute the command
+	cmd := exec.Command(command[0], command[1:]...)
+	// setup stdin, stdout and stderr
+	cmd.Stdin = c.Tty()
+	cmd.Stdout = c.Tty()
+	cmd.Stderr = c.Tty()
```

Comment:
We are setting them all to the same thing `c.Tty()`? How does this work? 🤔

Reply:
I am not completely sure how it works, this was something I got from the [Netflix/go-expect](https://github.com/Netflix/go-expect/blob/master/README.md) example.

Reply:
`c.Tty()` represents the master side of a pty (known also as ptm). Writes to the ptm is read from the slave side of a pty (known also as pts). Reads from the ptm is writes to the pts.